conf/testOnlyDoNotUseInAppConf.routes

```diff
@@ -16,4 +16,5 @@ GET        /protect-tax-info/test-only/successful
 GET        /users-groups-search/test-only/users/:credId                                                       uk.gov.hmrc.taxenrolmentassignmentfrontend.controllers.testOnly.TestOnlyController.usersGroupSearchCall(credId)
 GET        /protect-tax-info/test-only/auth/enrolments                                                        uk.gov.hmrc.taxenrolmentassignmentfrontend.controllers.testOnly.TestOnlyController.enrolmentsFromAuth()
 GET        /sa/test-only/start                                                                                uk.gov.hmrc.taxenrolmentassignmentfrontend.controllers.testOnly.TestOnlyController.successfulSACall
++nocsrf
```

Comment:
is this typo?

Reply:
No it is to stop the csrf token being required in the request